src/client/index.ts

```diff
@@ -1,12 +1,73 @@
 /**
```

Comment:
Needs a capitalization consistency pass for notification/Notifications in comments

Reply:
Will do. But should pick one first... Any preference?

My thoughts is 'notification', unless referring to either the `Notification` type, or the "Notifications service", rather than "a notification".

Reply:
👍

Reply:
After discussion, updated with the above approach.

Reply:
Also agreed on "Notification Center". Do Ctrl+Shift+F on this in the project and there's a few places to change

Reply:
(We also need a quote mark consistency pass, but beyond being able to worry about that right now)

Reply:
Have updated 'Notification Center' throughout, and did a quick quote-pass through the bits of the client that are visible in the API docs.